nestkernel/connection_manager.cpp

```diff
@@ -975,22 +971,6 @@ nest::ConnectionManager::data_connect_single( const index source_id,
   }
 }
 
-/**
- * Connect, using a dictionary with arrays.
- * The connection rule is based on the details of the dictionary entries source
- * and target.
- * If source and target are both either a GID or a list of GIDs with equal size,
- * then source and target are connected one-to-one.
- * If source is a gid and target is a list of GIDs then the sources is
- * connected to all targets.
- * If source is a list of GIDs and target is a GID, then all sources are
- * connected to the target.
- * At this stage, the task of connect is to separate the dictionary into one
- * for each thread and then to forward the connect call to the connectors who
- * can then deal with the details of the connection.
- *
- * @note This method is used only by DataConnect.
- */
```

Comment:
did you remove this comment completely or move it somewhere else?

Reply:
It was almost a duplication of what was already in the h-file, I just moved `@note`. And it's better to generally keep documentation in the h-files anyway.